pom.xml

```diff
@@ -2,14 +2,14 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.xebialabs.jira.addons</groupId>
+    <groupId>org.venkatk.jira.addons</groupId>
```

Comment:
Hi Ravan ..Thanks for the reply.. Am sorry I was not aware that further changes that I made into my forked repo to publish this to atlassian marketplace, would also be showing up in the pull request that I issued in XL Community repo.. I thought that it will only be the specific commit for which I had created the pull request, will be pulled in on to your repo.. Now I understand this and have already submitted the plugin to Altassian market place with my forked repo and awaiting Atlassian's approval